app/src/test/java/org/fossasia/openevent/app/presenter/AttendeePresenterTest.java

```diff
@@ -122,6 +122,8 @@ public void shouldLoadAttendeesSuccessfully() {
 
         attendeesPresenter.loadAttendees(false);
 
+        Collections.sort(attendees);
```

Comment:
`inOrder.verify(attendeesView).showAttendees(attendees);`

in this call `attendees` must be sorted because now we are sorting attendees before passing to the view in `AttendeesPresenter`